Yoast/wordpress-seo
# Limited max width of images from rich term descriptions in admin columns

## Summary

This PR can be summarized in the following changelog entry:

Fixed broken image in term description columns.

## Relevant technical choices:

* added style for `max-width` of images.

## Test instructions

This PR can be tested by following these steps:

1. Add large image to a term description
2. Observe that it doesn't break out of column in list view

Fixes #3644
